backend/app/resources.py

```diff
@@ -1,25 +1,41 @@
+import sys
+
+from flask_jwt_extended import create_access_token, create_refresh_token, jwt_required, jwt_refresh_token_required, \
+    get_jwt_identity, get_raw_jwt
 from flask_restful import Resource, marshal_with, marshal
 
-from app.marshallers import item_marshaller, category_marshaller, subcategory_marshaller
-from app.models import Item, Category, Subcategory
+from app.exceptions import IntegrityException, CreateTokenException
+from app.marshallers import item_marshaller, category_marshaller, subcategory_marshaller, user_marshaller
+from app.models import Item, Category, Subcategory, User, RevokedTokenModel
 from app.parsers import item_parser, subcategory_parser, category_parser, creating_item_parser, \
-    creating_category_parser, creating_subcategory_parser
+    creating_category_parser, creating_subcategory_parser, user_parser, login_user_parser
 from app.repositories import Repository
 from init_app import db
 
 
+def create_tokens(username):
+    try:
+        access_token = create_access_token(identity=username)
+        refresh_token = create_refresh_token(identity=username)
+        return access_token, refresh_token
+    except Exception:
```

Comment:
don't except Exception, is similar to empty except... What kind of error do you want to catch?